front/src/common/IntervalsEditor/IntervalsEditorToolButtons.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { ReactI18NextChild, useTranslation } from 'react-i18next';
+import { IoIosAdd } from 'react-icons/io';
+import { TbArrowsHorizontal, TbScissors } from 'react-icons/tb';
+import { BsFillTrashFill } from 'react-icons/bs';
+import { INTERVALS_EDITOR_TOOLS, IntervalsEditorTool, IntervalsEditorToolsConfig } from './types';
+
+const ToolButton = ({
+  selectedTool,
+  tool,
+  title,
+  icon,
+  toggleSelectedTool,
+}: {
+  selectedTool: IntervalsEditorTool | null;
+  tool: IntervalsEditorTool;
+  title: string;
+  icon: ReactI18NextChild;
+  toggleSelectedTool: (tool: IntervalsEditorTool) => void;
+}) => (
+  <button
+    className={`${selectedTool === tool ? 'btn-selected' : 'btn'} btn-sm btn-outline-secondary`}
+    type="button"
+    title={title}
+    onClick={() => {
+      toggleSelectedTool(tool);
+    }}
+  >
+    {icon}
+  </button>
+);
+
+type ToolButtonsProps = {
+  selectedTool: IntervalsEditorTool | null;
+  toolsConfig: IntervalsEditorToolsConfig;
+  toggleSelectedTool: (tool: IntervalsEditorTool) => void;
+};
+
+const ToolButtons = ({ selectedTool, toolsConfig, toggleSelectedTool }: ToolButtonsProps) => {
+  const { t } = useTranslation();
```

Comment:
as above, setting "common" here may load this json only (not sure)

Reply:
Done !